c2corg_ui/templates/helpers/view.html

```diff
@@ -234,9 +234,32 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
       <div class="associated-documents collapse in" id="associated-books">
         <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
           <app-card app-card-doc="doc"></app-card>
-          % if type == 'books':
-              <app-delete-association child-doctype="books" parent-id="${document['document_id']}" child-id="doc.document_id"
-                ng-if="${test_association_rights(document)}"></app-delete-association>
+          % if type == 'books' and document['doctype'] != 'xreports':
+            <app-delete-association child-doctype="books" parent-id="${document['document_id']}" child-id="doc.document_id"
+              ng-if="${test_association_rights(document)}"></app-delete-association>
+          % endif
+        </div>
+      </div>
+    </section>
+</%def>
+
+<%def name="show_associated_xreports(document)">\
+  <%
+    xreports = document['associations']['xreports']
+    associations = 'detailsCtrl.documentService.document.associations.' + type
+  %>
+   ${add_seo_association_links(xreports, 'xreports')}
+    <section ng-show="${associations}.length > 0">
+      <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#associated-xreports">
+        <span class="glyphicon glyphicon-map-marker"></span>
+        <span translate>Associated xreports</span><span class="glyphicon glyphicon-menu-down"></span>
+      </h3>
+      <div class="associated-documents collapse in" id="associated-xreports">
+        <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
+          <app-card app-card-doc="doc"></app-card>
+          % if type == 'xreports' and document['doctype'] != 'xreports':
+            <app-delete-association child-doctype="xreports" parent-id="${document['document_id']}" child-id="doc.document_id"
+              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
```

Comment:
I don't think that any authenticated user should be able to remove xreports associations.
It's probably the same than for other personal documents like articles or images. See for instance https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/helpers/view.html#L199-L215

That could be fine-tuned after golive but at least change the ``ng-if`` condition so that only moderators can remove the association.